### ISSUE TITLE
[2.0.1 regression] ModuleNotFoundError: No module named 'novelwriter.gui'

### ISSUE THREAD

**Issue by yurivict** [vkbo/novelWriter#1277]
```
$ novelwriter
Traceback (most recent call last):
  File "/usr/local/bin/novelwriter", line 33, in <module>
    sys.exit(load_entry_point('novelWriter==2.0.1', 'gui_scripts', 'novelwriter')())
  File "/usr/local/lib/python3.9/site-packages/novelwriter/__init__.py", line 239, in main
    from novelwriter.guimain import GuiMain
  File "/usr/local/lib/python3.9/site-packages/novelwriter/guimain.py", line 41, in <module>
    from novelwriter.gui.theme import GuiTheme
ModuleNotFoundError: No module named 'novelwriter.gui'
```

**Comment by vkbo** [comment]
Eh, I just fixed that in #1272. How did you install it?

**Comment by yurivict** [comment]
> How did you install it?

FreeBSD port.

**Comment by vkbo** [comment]
All the Linux packages in the release had this error, but they have been updated both here and on PyPi and Launchpad. I don't manage the FreeBSD port. It is probably still using the broken version.

**Comment by vkbo** [comment]
Ah, wait, that's you, right? I can make a new tag if that helps. There was no need for the other releases as it was enough to make a new build.

**Comment by vkbo** [comment]
I initially removed the __init__.py files in all subfolders and instead used direct imports everywhere to avoid repeated issues with circular imports. However, the setuptools build then proceeded to ignore all those folders, so I added back empty ones and rebuilt the wheel and deb files. They only got a build number bump though. I didn't do a full new release.

If that doesn't work with the port, please let me know if there is anything I can do to fix it. Preferably without having to do a full new release. Will a 2.0.1-post1 tag do the trick?

**Comment by yurivict** [comment]
Does the latest release include the fix?

**Comment by vkbo** [comment]
No, I just added the dummy files and rebuilt those specific packages. The files were merged after the tag. Commit `347266a9bdf06ae65fef558c0918a2dfa6a84a49` will build.

**Comment by vkbo** [comment]
Does the port pull release tags? It would be useful to know so I can avoid breaking it in the future. I was only considering the PyPi and Launchpad packages here as they both have mechanisms for dealing with this without a new release.

I can also just make the 2.0.2 release. There is also bug #1274 that needs fixing. It will take me about an hour.

**Comment by yurivict** [comment]
FreeBSD ports can read individual tags and revisions.
But they are usually updated based on new release tags.

**Comment by vkbo** [comment]
Ok, good to know. I will keep this in mind. I just fixed #1274, so I can make a new release. That bug annoyed me anyway.

**Comment by yurivict** [comment]
2.0.2 works. Thank you.
